platform/xbox/crt0.c

```diff
@@ -6,15 +6,32 @@
 
 extern void _PDCLIB_xbox_run_crt_initializers();
 extern int main (int argc, char **argv);
+extern mtx_t _PDCLIB_filelist_mtx;
+extern struct _PDCLIB_file_t * stdin;
+extern struct _PDCLIB_file_t * stdout;
+extern struct _PDCLIB_file_t * stderr;
 
 static int main_wrapper ()
 {
     _PDCLIB_xbox_tss_init();
 
+    mtx_init(&_PDCLIB_filelist_mtx, mtx_plain);
+    mtx_init(&stdin->mtx, mtx_recursive);
+    mtx_init(&stdout->mtx, mtx_recursive);
+    mtx_init(&stderr->mtx, mtx_recursive);
```

Comment:
nit: I wonder if this should be moved into a separate function, for users who don't use crt0 (my DXT code for example; but also some of my other lower-level project).

Reply:
I pushed an update that does this.